packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/index.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Connect your SCIM API service to Okta
-excerpt: Create and configure SCIM integrations, and check the attributes and their corresponding mappings in the Okta console.
+excerpt: Create a private SCIM integration instance. Configure and check the attributes and their corresponding mappings in the Okta Admin Console.
```

Comment:
```suggestion
excerpt: Create an instance of a private SCIM integration. Configure and check the attributes and their corresponding mappings in the Okta Admin Console.
```
Suggestion: Clarify that it isn't an "integration instance" but rather and instance of an integration.